tests/simtel/test_simtel_objects.py

```diff
@@ -724,3 +724,28 @@ def test_2032():
             assert 'n_times' in d
             assert 'pixel_ids' in d
             assert 'trigger_times' in d
+
+
+def test_2033():
+    from eventio.simtel.objects import PixelMonitoring
+
+    with EventIOFile('tests/resources/type2033.simtel.zst') as f:
+        objects = yield_n_and_assert(f, PixelMonitoring, n=19)
+        tel_id = 0
+        for tel_id, o in enumerate(objects, start=1):
+            assert o.header.id == tel_id
+            data = o.parse()
+            assert data.keys() == {
```

Comment:
The `fadc_amp_lg` key would not be there if the test file does not have 2 gains. Maybe it's better to make the test more general?

Reply:
To limit the size of the test files, I produced a la palma run, but you are right, we should have both cases.